pkg/apis/troubleshoot/v1beta2/hostanalyzer_shared.go

```diff
@@ -60,6 +60,12 @@ type TimeAnalyze struct {
 	Outcomes      []*Outcome `json:"outcomes" yaml:"outcomes"`
 }
 
+type TLSAnalyze struct {
```

Comment:
Should we name this HostTLSAnalyze to keep it consistent with the others?

Also, similar to collector_shared, you'll need to add this to the `HostAnalyze` struct as well.

Reply:
🤦

Reply:
> Should we name this HostTLSAnalyze to keep it consistent with the others?

very few of the others have 'host' in the name - just 4 of 25